cypress/components/split-button/split-button.cy.js

```diff
@@ -482,17 +514,11 @@ context("Tests for SplitButton component", () => {
   it("should have the expected border radius and focus styling on main and toggle buttons", () => {
```

Comment:
```suggestion
  it("should have the expected border radius on main and toggle buttons", () => {
```

**comment**: as on multi-action button

Reply:
Changed as per comment